### PR TITLE
Add play liked songs options

### LINKED ISSUE #7473: [Spotify Player] Play liked songs with option to shuffle
### Extension

https://www.raycast.com/mattisssa/spotify-player

### Description

Currently I can't find a way to play my liked songs. I can search for individual liked songs but that will play the song in isolation rather than playing through all of them. Additionally I'd like the option to shuffle play whatever I'm about to choose.

### Who will benefit from this feature?

Anyone who regularly listens to their liked songs.

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by bkeys818**
## Description

Adds a _liked song_ playlist in the library and search commands.

Closes #7473 and #5996.

## Screencast

Screenshot 2023-09-29 at 6 48 48 PM

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by bkeys818** [comment]
Anyone had a chance to check this out? Not sure if it's worth merging, but I'd love to get some other opinions.

**Comment by peduarte** [comment]
I'll get on top of this asap

File: extensions/spotify-player/src/api/play.ts [hunk 1]
```diff
@@ -25,7 +25,7 @@ export async function play({ id, type, contextUri }: PlayProps = {}) {
 
   try {
     if (!type || !id) {
-      await spotifyClient.putMePlayerPlay();
+      await spotifyClient.putMePlayerPlay({ context_uri: contextUri });
```
Comment (peduarte) [inline extensions/spotify-player/src/api/play.ts:28]: is this safe? what if `contextUri` is undefined?

**Comment by peduarte** [comment]
A comment, then happy to merge after conflicts are resolved.

File: extensions/spotify-player/src/api/play.ts (hunk 1 above)
Comment (bkeys818) [reply extensions/spotify-player/src/api/play.ts:28]: I just tested the uses of `play` that don't provide `contextUri`, and everything seems to work fine. I think calling `putMePlayerPlay` when `contextUri` is undefined is no different than not providing it at all.

**Comment by peduarte** [comment]
Sorry, there's been a few other changes coming in, so there's another conflict. Can you resolve that please, and lets get it in! 🚀

**Comment by peduarte** [comment]
Some errors

**Comment by peduarte** [comment]
👀

**Comment by bkeys818** [comment]
My bad! I was working with the wrong version of prettier.

**Comment by bkeys818** [comment]
Ok, try three

**Pull request merged**